master/getting-started/kubernetes/installation/hosted/index.md

```diff
@@ -92,14 +92,6 @@ Calico's manifests assign its components one of two service accounts.
 Depending on your cluster's authorization mode, you'll want to back these
 ServiceAccounts with the neccessary permissions.
```

Comment:
Is this para still necessary?  As it stands it doesn't really say anything about how to do that.